jvm/src/main/scala/io/glutenproject/execution/HashJoinExecTransformer.scala

```diff
@@ -20,14 +20,14 @@ package io.glutenproject.execution
 import com.google.common.collect.Lists
 import com.google.protobuf.{Any, ByteString}
 import io.glutenproject.GlutenConfig
-import io.glutenproject.execution.HashJoinLikeExecTransformer.{makeAndExpression, makeEqualToExpression}
+import io.glutenproject.execution.HashJoinLikeExecTransformer.{makeAndExpression, makeEqualToExpression, makeIsNullExpression}
```

Comment:
use `HashJoinLikeExecTransformer.makeAndExpression`, and remove these imports?
`makeIsNullExpression` is un-used import in this code

Reply:
Fixed.